guides/plugins/plugins/administration/extending-webpack.md

```diff
@@ -0,0 +1,37 @@
+# Extending Webpack
+
+## Overview
+
+The Shopware 6 Administration uses [Webpack](https://webpack.js.org/) as a static module bundler.
+The default configuration should work for most plugins, 
```

Comment:
I would say that if you don´t have needs for any special changes then you don´t need to bother with webpack at all. Your plugin will just work.